server/game/cards/02.6-TS/Jhogo.js

```diff
@@ -20,13 +20,13 @@ class Jhogo extends DrawCard {
 
     getNumberOfDeadDefendingCharacters() {
         var deadDefenders = [];
-        this.game.currentChallenge.defendingPlayer.deadPile.each(card => {
+        for(const card of this.game.currentChallenge.defendingPlayer.deadPile) {
```

Comment:
Unrelated to changes in this PR, but this check is outright wrong. Should literally just be `deadPile.length`, regardless of uniques in the dead pile.